src/esp/assets/PTexMeshData.cpp

```diff
@@ -602,35 +616,56 @@ void PTexMeshData::uploadBuffersToGPU(bool forceReload) {
                                   currentMesh->abo);
     currentMesh->abo.setData(adjFaces[iMesh],
                              Magnum::GL::BufferUsage::StaticDraw);
+
+    // experiment code (may not work):
     currentMesh->mesh.setPrimitive(Magnum::GL::MeshPrimitive::LinesAdjacency)
         .setCount(currentMesh->ibo.size() / 2)
         .addVertexBuffer(currentMesh->vbo, 0, gfx::PTexMeshShader::Position{})
         .setIndexBuffer(currentMesh->ibo, 0,
                         Magnum::GL::MeshIndexType::UnsignedInt);
   }
 
+  // load atlas data and upload them to GPU
+
+  std::cout << "loading atlas textures: " << std::endl;
   for (size_t iMesh = 0; iMesh < renderingBuffers_.size(); ++iMesh) {
-    const std::string rgbFile =
-        atlasFolder_ + "/" + std::to_string(iMesh) + "-color-ptex.rgb";
-    if (!io::exists(rgbFile)) {
-      ASSERT(false, "Can't find " + rgbFile);
-    }
+    const std::string rgbFile = Corrade::Utility::Directory::join(
+        atlasFolder_, std::to_string(iMesh) + "-color-ptex.rgb");
+
+    ASSERT(io::exists(rgbFile), Error : Cannot find the rgb file);
```

Comment:
There's [CORRADE_ASSERT()](https://doc.magnum.graphics/corrade/Assert_8h.html#a83f7361970111951c88f1564a4f148e8) btw, which could make `esp/logging.h` obsolete -- and it supports printing everything that [Utility::Debug](https://doc.magnum.graphics/corrade/classCorrade_1_1Utility_1_1Debug.html) can :) 

```suggestion
    CORRADE_ASSERT(io::exists(rgbFile), "Error : Cannot find the rgb file");
```

Also, for `io::exists` there's [Directory::exists()](https://doc.magnum.graphics/corrade/namespaceCorrade_1_1Utility_1_1Directory.html#a0299a9e5b18b4187615c2fd8e256d586) ... and and and

... 🤔 should I open a PR moving everything from `esp/logging.h` and `io/io.h` to builtin Corrade functionality? :smile: (cc: @msavva @erikwijmans)